pages/national_resources/cy_resources.md

```diff
@@ -53,12 +52,22 @@ national_resources:
 <!---Please focus on resources that are relevant for the whole country for life sciences--->
 
 ## Introduction 
-<!---General RDM considerations for your country, how to deal with RDM on a national level--->
+This page provides a general overview of national resources and initiatives on research data management (RDM) in Cyprus
 
 ## Funders
+H2020-WIDESPREAD-01-2018-2019: Teaming Phase 2, for €15mi for 7 years. €15mi by the Government of Cyprus and another €8mi by the  University of Cyprus. 
+The Research and Innovation Foundation (RIF) is the national authority in charge of supporting and promoting research, technological development and innovation in Cyprus.
 
 ## Regulations
-<!--- Ethical and legal regulations in the country, committees, etc. --->
+Cyprus National Bioethics Committee (document EEBK3). 
+All medical records and biological material are kept under secure conditions at a specially prepared 
+Biobank suite with absolute confidentiality and protection of sensitive data of personal character, 
```

Comment:
Also, a link to Biobank website for Cyprus needs to be linked here.

Reply:
Added link, thanks!